cli/tests/unit/net_test.ts

```diff
@@ -65,6 +65,44 @@ unitTest(
   },
 );
 
+unitTest(
+  { ignore: Deno.build.os === "windows", perms: { read: true } },
+  function netUnixListenWritePermission(): void {
+    try {
+      const filePath = Deno.makeTempFileSync();
+      const socket = Deno.listen({
+        path: filePath,
+        transport: "unix",
+      });
+      assert(socket.addr.transport === "unix");
+      assertEquals(socket.addr.path, filePath);
+      socket.close();
+    } catch (e) {
+      assert(!!e);
+      assert(e instanceof Deno.errors.PermissionDenied);
+    }
```

Comment:
@wperron this is a bit brittle - if function doesn't throw test will pass anyway. There's a helper assert called `assertThrows` that could be used for this purpose.

Reply:
I'll open a new PR for that